src/payment/strategies/clearpay/clearpay-window.ts

```diff
@@ -0,0 +1,5 @@
+import ClearpaySdk from './clearpay-sdk';
+
+export default interface ClearpayWindow extends Window {
+    AfterPay: ClearpaySdk;
```

Comment:
```suggestion
    ClearPay: ClearpaySdk;
```

Reply:
The global variable in the clearpay's script is called Afterpay for this reason I leave this name.